### DISCUSSION TITLE
Open PDF in new tab instead of downloading it?

### DISCUSSION THREAD

**Discussion by sohnemann** [requarks/wiki#5054] [Help / Questions]
Hi everyone,

At the moment when I click on a link to a local pdf file it downloads it.
Is there any way to open the pdf in a new tab for viewing?

I'm not talking about embedding, just showing it in another tab.
What I've read so far is that the Response to the GET-Request needs to contain the correct **Content-Type** and **Content-Disposition**,
something that can only be done server-side.

Any help would be much appreciated.

Cheers,
sohnemann

**Comment by Phil-hostwiki** [comment]
Wiki.js renders files other than images as downloadable attachments. https://github.com/Requarks/wiki/blob/8d420759f886b344cb1d212e7a03b8ceac44ddbb/server/models/assets.js#L175-L178

There is also the `forceDownload` set to `true` in the app data.yml file.
https://github.com/Requarks/wiki/blob/b7f50633f3b455471f3db93a89cb4bbf3a528b77/server/app/data.yml#L86
